doc/references.bib

```diff
@@ -0,0 +1,80 @@
+@book{Ahrens2012,
+    author = {Ahrens, J.},
+    title = {{Analytic Methods of Sound Field Synthesis}},
+    publisher = {Springer},
+    address = {Berlin Heidelberg},
+    year = {2012},
+    doi = {10.1007/978-3-642-25743-8}
+}
+@book{Moser2012,
+    author = {M\"{o}ser, M.},
```

Comment:
If you switch to the `alpha` style, you should write the proper letter `ö` here (and probably use the UTF-8 setting for the `bibliography` directive). Otherwise it doesn't seem to be parsed correctly.